solutions/blocks/awards-search/awards-search.css

```diff
@@ -102,8 +102,8 @@ main .section .awards-search .awards-results-container .award-item-container h2
     width: 1.5rem;
     height: 1.3125rem;
     /* stylelint-disable-next-line property-no-vendor-prefix */
-    -webkit-mask: url('/icons/arrow-right.svg');
-    mask: url('/icons/arrow-right.svg');
+    -webkit-mask: url('../../icons/arrow-right.svg');
```

Comment:
I would use `/solutions/icons/arrow-right.svg` in order to have all the paths the same in the repo
I would suggest to use the root relative path, instead of the absolute one